packages/near-membrane-base/src/membrane.ts

```diff
@@ -1099,77 +1099,6 @@ export function init(
     // future optimization: hoping that proxies with frozen handlers can be faster
     ObjectFreeze(BoundaryProxyHandler.prototype);
 
-    function linkGlobalThis(foreignGlobalThisPointers: Pointer) {
```

Comment:
this membrane implementation was not very generic, it made some assumptions about reflection of intrinsics and unforgeables, which was not pretty, but I finally cracked this one. This is now a very generic membrane implementation that can work with callable boundary, finally!